Apps/WebClient/src/NewClientApp/src/stores/notification.ts

```diff
@@ -0,0 +1,128 @@
+﻿import { defineStore } from "pinia";
+import { ref } from "vue";
+import { ResultError } from "@/models/errors";
+import { LoadStatus } from "@/models/storeOperations";
+import Notification from "@/models/notification";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { ILogger, INotificationService } from "@/services/interfaces";
+import { container } from "@/ioc/container";
+import { useUserStore } from "@/stores/user";
+import { ErrorSourceType, ErrorType } from "@/constants/errorType";
+import { useErrorStore } from "@/stores/error";
+
+export const useNotificationStore = defineStore("notification", () => {
+    const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
+    const notificationService = container.get<INotificationService>(
+        SERVICE_IDENTIFIER.NotificationService
+    );
+
+    const userStore = useUserStore();
+    const errorStore = useErrorStore();
+
+    const notifications = ref<Notification[]>([]);
+    const statusMessage = ref("");
+    const error = ref<ResultError>();
+    const status = ref(LoadStatus.NONE);
+
+    function setRequested() {
+        status.value = LoadStatus.REQUESTED;
+    }
+
+    function setNotifications(incomingNotifications: Notification[]) {
+        notifications.value = incomingNotifications;
+        error.value = undefined;
+        status.value = LoadStatus.LOADED;
+    }
+
+    function clearNotification(notificationId: string) {
+        notifications.value = notifications.value.filter(
+            (n: Notification) => n.id != notificationId
```

Comment:
```suggestion
            (n) => n.id != notificationId
```
Type specification should not be necessary.